pkg/ibm/datamodel/data_model.go

```diff
@@ -64,14 +64,20 @@ func (res *VPC) UnmarshalJSON(data []byte) error {
 	}
 	res.VPC = *asObj
 
-	err = json.Unmarshal(asMap["address_prefixes"], &res.AddressPrefixes)
-	if err != nil {
-		return err
+	val, ok := asMap["address_prefixes"]
+	if ok {
+		err = json.Unmarshal(val, &res.AddressPrefixes)
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
		if err = json.Unmarshal(val, &res.AddressPrefixes) ; err != nil {
			return err
		}
```

shorter syntax.. change also in the other place